sdk/storage/Azure.Storage.DataMovement.sln

```diff
@@ -19,6 +19,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.DataMovement.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.DataMovement", "Azure.Storage.DataMovement\src\Azure.Storage.DataMovement.csproj", "{B152F963-EB83-444E-8B39-4E464C48C322}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.DataMovement.Blobs.Stress", "Azure.Storage.DataMovement.Blobs\stress\src\Azure.Storage.DataMovement.Blobs.Stress.csproj", "{1AD25785-5234-46E3-BF0C-A04B76DDF9E1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{C1C88D7D-2513-4A33-BB8B-11FCF2DE301A}"
+EndProject
```

Comment:
I noticed the other day that I think this also needs Common?

Reply:
Yeah I can add that. It's strange, I'm running into the whole "it works on my machine" but apparently no one elses

Reply:
Done added below